IDE/Espressif/ESP-IDF/examples/wolfssl_benchmark/main/helper.c

```diff
@@ -21,13 +21,109 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <wolfssl/wolfcrypt/settings.h>
 
 #include "sdkconfig.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "esp_log.h"
+#include "nvs_flash.h"
+
+#define WOLFSSL_BENCH_ARGV                 CONFIG_BENCH_ARGV
+#define WOLFSSLBENCHMARK_TASK_NAME         "wolfsslbenchmark_name"
+#define WOLFSSLBENCHMARK_TASK_WORDS        10240
+#define WOLFSSLBENCHMARK_TASK_PRIORITY     8
+
+/* proto-type */
+extern void wolf_benchmark_task();
+extern int benchmark_init();
+extern int benchmark_test(void *args);
+
+#ifdef WOLFSSL_ESPWROOM32SE
+const static char* TAG = "wolfbenchmark";
+#else
+const char* TAG = "wolfbenchmark";
+#endif
 
-#define WOLFSSL_BENCH_ARGV              CONFIG_BENCH_ARGV
 
 char* __argv[22];
 
+#if defined(WOLFSSL_ESPWROOM32SE) && defined(HAVE_PK_CALLBACKS) \
+                                  && defined(WOLFSSL_ATECC508A)
+
+#include "wolfssl/wolfcrypt/port/atmel/atmel.h"
+
+int atcatls_set_callbacks(struct WOLFSSL_CTX* ctx);
```

Comment:
This function prototype is in`wolfssl/wolfcrypt/port/atmel/atmel.h`, so you should not need this here.

Reply:
Yes, thanks. Removed.